api/src/lib/settings/index.ts

```diff
@@ -73,7 +73,9 @@ export async function getAllSettings(
 
   const mappedToSchema = results.reduce<Record<string, string>>(
     (acc, setting) => {
-      acc[setting.key] = setting.value ? JSON.parse(setting.value) : undefined;
+      acc[setting.key] = setting.value
+        ? safeParseJson(setting.value)
+        : undefined;
```

Comment:
I don't think this is required after running the migration. The migration will clear the settings and the api endpoint for upserting settings validates the data with the `SettingsSchema`

If we have breaking settings changes we should write migrations for those?

Reply:
yeah it's not but also if some non-json sneaks its way into one field in the db, then the entire settings route will crash at this point 😬 

i know defensive coding isn't always the best but! seemed better than the alt